tracker/src/plausible.js

```diff
@@ -167,4 +167,28 @@
   {{#if (any outbound_links file_downloads tagged_events)}}
   {{> customEvents}}
   {{/if}}
+
+  {{#if live_view}}
+    window.addEventListener('phx:navigate', info => trigger('pageview', {u: info.detail.href}))
+    
+    const events = ["phx:page-loading-start", 
```

Comment:
We should support all live view events bindings:
https://hexdocs.pm/phoenix_live_view/bindings.html